fc/parsing/CompactSyntaxParser.py

```diff
@@ -726,16 +513,15 @@ def _expr(self):
     ######################################################################
 
     class SetTimeUnits(BaseAction):
-        def _xml(self):
-            return P.setIndependentVariableUnits(**self.TransferAttrs('units'))
+        pass
```

Comment:
With all these model interface sections, it would be worth storing the useful properties as member variables in some fashion, or at least documenting what is available, to assist with replacing the protocol parts of pycml later.

Reply:
For instance, the `TransferAttrs` function is only needed by XML generation. But it'd probably be handy to keep a renamed version that does the extraction as a dict, and save that dict as a member var.